src/modules/UI/Wallets/reducer.js

```diff
@@ -182,6 +182,22 @@ export const walletEnabledTokens = (state: any = {}, action: Action) => {
   return state
 }
 
+export const walletLoadingProgress = (state: Object = {}, action: Action) => {
+  if (!action.data) return state
+  const {type, data} = action
+  switch (type) {
+    case ACTION.INSERT_WALLET_IDS_FOR_PROGRESS:
```

Comment:
if you can grab the payload from the login dispatch...
`case Constants.AccountInitComplete:`

Reply:
additionally, do any data munging here